VaticanBAV/et/BAVet123.xml

```diff
@@ -0,0 +1,233 @@
+        <fileDesc>
+            <titleStmt>
+                <title>Śǝna fǝṭrat, Ṣalota māʿǝdd, Tǝrgʷāme fidal</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MV"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0003BAV"/>
+                        <collection>Aethiopici</collection>
+                        <idno>Aeth. 123</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="3r" to="66r"/>
+                            <title type="complete" ref="LIT2314senafe"/>
+                            <textLang mainLang="am" otherLangs="gez"/>
+                            <explicit>
+                                ወኤልዩድኒ፡ ወለደ፡ አልአዛርሃ፡ ወአልአዛርኒ፡ ወለደ<supplied reason="omitted">፡</supplied> ቅስራሃ፡ ወማትያንሃ፡ ማትያንም፡ ያዕቆ<supplied reason="omitted">ብ</supplied>ሃ፡
+                                ወለደ፡ ያፅቆብኒ፡ ወለደ፡ ዮሴፍሃ፡ <choice><sic>ፈሃራሃ፡</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ፈሃሪሃ፡</corr></choice> ለማርያም፡ ወዮሴፍ፡ 
+                                ወለዶ፡ ለያዕቆብ፡ ኤጲስ፡  ቆጶስ፡ ዘኢየሩሳሌም፡ ወያ<surplus>ያ</surplus>ዕቆብ፡ <choice><sic>ወለደ፡</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ወለዶ፡</corr></choice> 
+                                ለኢያቄም፡ ወኢያቄም፡ ወለዳ፡ ለማርያም፡ ማርያምኒ፡ ወለደቶ፡ ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ዝውእቱ<supplied reason="omitted">፡</supplied> ኍልቈ፡ 
+                                ልደተ፡ ሥጋሁ፡ ለክርስቶስ፡ ፷ወ፬ ትውልድ<supplied reason="omitted">፡</supplied> ፶፻ወ፭፻ ዓመት፡ ይህ<supplied reason="omitted">፡</supplied> ነው፡ 
+                                ስብሐት፡ ለእግዚአብሔር፡ ዘአፈጸመኒ፡ በሰላም፡ <gap reason="omitted" resp="PRS4805Grebaut PRS9530Tisseran"/> ይምሀረነ<supplied reason="omitted">፡</supplied> 
+                                እግዚአብሔር፡ አሜን፡ ወአሜን፨
+                            </explicit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i2">
+                            <locus target="#67r"/>
+                            <title type="incomplete" ref="LIT7178SalotaMa"/>
+                            <textLang mainLang="gez"/>
+                            <incipit>
```

Comment:
all incipit and explicit elements need to contain xml:lang="gez" (even if it is already stated in textLang)